aws_cloud_cost/main.yaml

```diff
@@ -0,0 +1,164 @@
+# version: 0.0.1
+AWSTemplateFormatVersion: 2010-09-09
+Description: Datadog Cloud Cost Management Setup
+Parameters:
+  DDCloudCostBucketName:
+    Type: String
+    Description: The name of the S3 bucket for storing the cost and usage reports.
+    MinLength: 1
+  DDCloudCostBucketRegion:
+    Type: String
+    Description: The AWS region where the S3 bucket is located.
+    MinLength: 1
+  DDCloudCostBucketPrefix:
+    Type: String
+    Description: The prefix to be added to the cost and usage report files in the S3 bucket.
+    MinLength: 1
+  DDCloudCostReportName:
+    Type: String
+    Description: The name of the cost and usage report.
+    MinLength: 1
+  DatadogIntegrationRole:
+    Type: String
+    Description: The name of the IAM role that has been integrated with Datadog.
+    MinLength: 1
+  CreateCloudCostReport:
+    Type: String
+    Default: true
+    AllowedValues:
+      - true
+      - false
+    Description: Should a new Cost and Usage Report be created, or use an existing one.
```

Comment:
Wording sounds a little weird. Not sure what's the best way to phrase it. Maybe something like: "Whether a new Cost and Usage Report is created, or an existing one is used."
Same for the CreateCloudCostBucket description below.